app/src/main/java/io/heckel/ntfy/db/Repository.kt

```diff
@@ -292,6 +292,16 @@ class Repository(private val sharedPrefs: SharedPreferences, private val databas
             .apply()
     }
 
+    fun getEnableUP(): Boolean {
+        return sharedPrefs.getBoolean(SHARED_PREFS_ENABLE_UP, true) // Enabled by default
+    }
+
+    fun setEnableUP(enabled: Boolean) {
+        sharedPrefs.edit()
+            .putBoolean(SHARED_PREFS_ENABLE_UP, enabled)
+            .apply()
+    }
```

Comment:
Could you call this `getUnifiedPushEnabled`? "UP" is obvious to us now, but it should be obvious to the casual reader of the code too